### DISCUSSION TITLE
Override DNS resolve results like dnsmasq

### DISCUSSION THREAD

**Discussion by CaledoniaProject** [DNSCrypt/dnscrypt-proxy#1810] [Q&A]
I'm trying to override DNS resolve results, like what /etc/hosts do.

But I went through the possible configuration options, it does not seem to be supported:
https://github.com/DNSCrypt/dnscrypt-proxy/blob/master/dnscrypt-proxy/config.go#L30

Can you add this feature? At least dnsmasq supports this: 

* address=/prod.example.com/10.0.0.2
* addn-hosts=/etc/dnsmasq.hosts

**Comment by ghost** [comment]
[Read me.](https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Cloaking)

**Comment by CaledoniaProject** [comment]
Awesome, this is way more advanced than dnsmasq
